src/Database/Mysql/Driver/Mysql.php

```diff
@@ -43,11 +43,19 @@ public function create(Tenant $tenant): bool
 
         event(new Events\Creating($tenant, $config, $this));
 
-        return $this->processAndDispatch(Events\Created::class, $tenant, [
-            'user'     => "CREATE USER IF NOT EXISTS `{$config['username']}`@'{$config['host']}' IDENTIFIED BY '{$config['password']}'",
-            'database' => "CREATE DATABASE `{$config['database']}`",
-            'grant'    => "GRANT ALL ON `{$config['database']}`.* TO `{$config['username']}`@'{$config['host']}'",
-        ]);
+        $qm = app(QueryManager::class);
```

Comment:
Also let's not use the app helper, but the facade :)

Reply:
Can you expand on this?  What facade?

Reply:
Just in general not a fan of the `app` helper and probably getting rid of it everywhere in a future PR.

Reply:
right, but what facade would you use instead?